lib/kredits.js

```diff
@@ -22,16 +22,16 @@ function capitalize(word) {
 
 class Kredits {
 
-  constructor(provider, signer, addresses) {
+  constructor(provider, signer, options = {}) {
     this.provider = provider;
     this.signer = signer;
 
     // by default we only need the registry address.
     // the rest is loaded from there in the init() function
-    this.addresses = addresses || {Registry: RegistryAddress[this.provider.chainId.toString()]}; // chaiID must be a string
-    this.abis = ABIS;
+    this.addresses = options['addresses'] || { Registry: RegistryAddress[this.provider.chainId.toString()] }; // chaiID must be a string
+    this.abis = options['abis'] || ABIS;
```

Comment:
You sneaked that one in but I ❤️ it,

Reply:
Typo. :)

Reply:
iPad...

Reply:
I meant in the code. Someone coding on an iPad? :D

Reply:
Ah you mean the `chaiID` 😆 ? Not yet coding on an iPad...
It was a bad idea to come back to Hamburg. Volley outside...

Reply:
:sun_with_face: